ckanext/ontario_theme/i18n/fr/LC_MESSAGES/ckanext-ontario_theme.po

```diff
@@ -639,33 +650,35 @@ msgid ""
 "ontario-data-catalogue\">the survey</a> and tell us how we’re doing. </p> "
 "</li> </ul>"
 msgstr "\n"
-"<h2 class=\"ontario-small\">\n"
+"<nav> <div class=\"ontario-page-navigation--full\"> <div class=\"ontario-"
+"page-navigation-content--full\">"
+"<h2>\n"
 "  Dans cette page\n"
 "</h2>\n"
-"<ol class=\"table-of-contents\">\n"
-"  <li>\n"
-"    <a href=\"#open-data\">Données ouvertes</a>\n"
+"<a class=\"ontario-show-on-focus\" href=\"#skip-to-main\">Sauter les liens de navigation de cette page</a>\n"
+"<ol class=\"ontario-page-navigation-list\">\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#open-data\">Données ouvertes</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#other-data\">Autres données</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#other-data\">Autres données</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#licensing\">Licence d’utilisation des données</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#licensing\">Licence d’utilisation des données</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#developers\">Ressources pour les concepteurs</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#developers\">Ressources pour les concepteurs</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#training-materials\">Documents de formation</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#training-materials\">Documents de formation</a>\n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#metadata\">Ce que nous vous disons à propos des données (métadonnées)?</a>   \n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#metadata\">Ce que nous vous disons à propos des données (métadonnées)?</a>   \n"
 "  </li>\n"
-"  <li>\n"
-"    <a href=\"#engagement\">Engagement</a>\n"
+"  <li class=\"ontario-page-navigation-list__item\">\n"
+"    <a class=\"ontario-page-navigation-item__link\" href=\"#engagement\">Engagement</a>\n"
```

Comment:
You will need the skip to main div in the French section of the .po file here as well.